Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml

```diff
@@ -4,7 +4,8 @@ commonfields:
 name: Active Directory Query v2
 display: Active Directory Query v2
 category: Data Enrichment & Threat Intelligence
```

Comment:
@anara123 @michalgold what are we going to do with the integration category? (should be `Identity and Access Management` in order to be taken into account in the dashboard)

Reply:
hard code it in the widget query for now, we'll handle this issue in phase2